### PR TITLE
Update sample vars and secrets and test doc

### PR DESCRIPTION
**Pull request by bogdando**
Add missing dependencies insntallation step into testing docs (based on the adoption hackathon demos recoreded)

Improve samples default values so that user can make less manual editions to start the test suits.

### CODE REVIEW

**Comment by bogdando** [comment]
still testing this

File: tests/vars.sample.yaml [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-source_ovndb_ip: CUSTOMIZE_THIS
+source_ovndb_ip: 192.168.122.100 #CUSTOMIZE_THIS
 
 # EDPM node IP
-edpm_node_ip: CUSTOMIZE_THIS
+edpm_node_ip: 192.168.122.100 #CUSTOMIZE_THIS
 
 # NTP servers list
 edpm_chrony_ntp_servers:
+  - clock.redhat.com
```
Comment (jistr) [inline tests/vars.sample.yaml:21]: I don't think we should default to this in public code... E.g. the upstream CI won't be able to reach that. Other than this LGTM, this PR is a nice UX improvement for local testing, thanks :+1:

Comment (bogdando) [reply tests/vars.sample.yaml]: commented it out, but it is crucial to use it when testing from the intranet, I can recall multiple problems related to missing that source from downstream tripleo background

**Review by jistr — APPROVED** [review]
Thanks!

**Review by fao89 — APPROVED** [review]

**Pull request merged**